lib/mauth/client/remote_authenticator.rb

```diff
@@ -62,12 +70,14 @@ def make_mauth_request(authentication_ticket)
       end
 
       def mauth_connection
-        require 'faraday'
-        require 'faraday_middleware'
-        @mauth_connection ||= ::Faraday.new(mauth_baseurl, faraday_options) do |builder|
-          builder.use MAuth::Faraday::MAuthClientUserAgent
-          builder.use FaradayMiddleware::EncodeJson
```

Comment:
I see, I am guessing this is super legacy

Reply:
not super legacy but not compatible with Faraday 2